roles/migrationcontroller/templates/controller.yml.j2

```diff
@@ -71,6 +71,10 @@ spec:
           value: webhook-server-secret
         - name: MIGRATION_REGISTRY_IMAGE
           value: {{ migration_registry_image_fqin }}
+{% if mig_controller_enable_cache|bool is true %}
```

Comment:
@jmontleon @djwhatle 

I think this is the thing that is causing the error here:

https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/konveyor_mig-operator/675/pull-ci-konveyor-mig-operator-master-operator-e2e/1395842958757990400/artifacts/operator-e2e/gather-extra/artifacts/pods/openshift-migration_migration-operator-7fc96f478f-h5qhx_operator.log

Reply:
Yes. `is true` and `is false` are builtin tests were not added to Jinja 2 until 2.11 and the RPM on ubi8 is 2.10.x. You need to use comparison operators like `==`, `!=`, etc. to satisfy downstream.

It works upstream because ansible-operator does a `pip install` upstream o install. This gets a more current version of Jinja2 thank is available via RPM on ubi8.

Reply:
See https://jinja.palletsprojects.com/en/3.0.x/templates/#comparisons vs https://jinja.palletsprojects.com/en/3.0.x/templates/#list-of-builtin-tests